### PR TITLE
docs: seed repository-wide collaborative review

### PR DESCRIPTION
**Pull request by yifeif-nv**
## Background

This draft intentionally touches all 117 tracked Markdown files so the full documentation surface can be reviewed collaboratively in one GitHub pull request. It includes the root README and policy files, component READMEs, repo-local skill documentation, and all 82 Markdown files under `website/`.

No prose or rendered documentation is changed yet. Each file receives one non-rendering anchor that provides a stable line for inline review threads.

## Exit Criteria

- All 117 tracked Markdown files appear in the pull-request diff.
- The anchors do not alter rendered documentation.
- Reviewers can open an inline thread on every file and replace anchors with substantive edits as review progresses.
- This draft is review scaffolding, not a merge-ready documentation update.

## Implementation

- Added an MDX-safe `{/* Collaborative review anchor. */}` to each of the 81 Docusaurus documents under `website/docs/`.
- Added an HTML `<!-- Collaborative review anchor. -->` to the other 36 tracked Markdown files.
- Kept the diff mechanical and uniform: 117 files, 234 insertions, and 0 deletions.

## Validation

- `git diff --check`: passed.
- Exact topology check against the tracked Markdown inventory: passed; 117/117 files are present and each file has exactly two added lines with zero deletions.
- `python3 tools/check_doc_file_references.py --strict website/docs`: passed; 81 documents and 220 references checked with zero findings.
- `python3 tools/model_ci.py validate`: passed; 78 model families validated.
- `python3 tools/test_impact.py --validate`: passed with pre-existing stale-allowlist warnings.
- Focused documentation-consumer tests: passed, 9/9.
- `npm run test:model-support`: passed, 1/1.
- `npm run build`: passed; 36 diagrams verified and the Docusaurus production build completed. This local run used Node v25.6.1; GitHub Pages CI uses Node 20.
- `python3 tools/legal_headers.py --check`: passed.

### Expected Legal check failure

`CONTRIBUTING.md` is deliberately included because the requested scope is every Markdown file. The Legal workflow byte-pins that file to SHA-256 `efc27920357d41bd5fe87108dbf35089b5bed1c90a05cf23577c9e1b52c698eb`; the review anchor changes it to `021d438321a3be5f29a870aa1b1c4c44be12a5a5f0c3f9f893403a631cb45c48`.

The pinned digest is intentionally not updated just to make CI pass. This draft should remain non-merge-ready until a human/legal owner decides how `CONTRIBUTING.md` should participate in the review.

## Notes For Future Readers

- Suggested starting order: `README.md`, `website/docs/intro.md`, `website/docs/learning-path.md`, then `website/docs/getting-started/`.
- Use each added anchor as the inline discussion point for that file.
- Remove or replace anchors as substantive review edits land.
- Do not update the canonical legal digest solely to green this review-seeding draft.

### CODE REVIEW

File: website/docs/getting-started/model-support.md [hunk 1]
```diff
@@ -168,3 +168,5 @@ inspect its family-owned profile TOML. The retained qualified profiles are
 exact product routes; because Source does not publish their former
 target-hardware runner, a Source checkout alone cannot reproduce their
 hardware qualification.
+
+{/* Collaborative review anchor. */}
```
Comment (nvluxiaoz) [inline website/docs/getting-started/model-support.md]: **[Claude Agent Review]** Several of these families — InternVL3/3.5, Qwen3.5/3.6, Qwen2.5-VL / Qwen3-VL, Phi-4-Multimodal, Nemotron-Omni, Qwen3-Omni — are already implemented and CI-qualified in TensorRT Edge-LLM, and TRTMC's one qualified runtime (Qwen3) already delegates to Edge-LLM. Since TRTMC currently carries parallel (unqualified) implementations of the same families, and given the plan to dispatch model support to Edge-LLM, could we clarify per family whether the intent is to delegate to Edge-LLM (as the Qwen3 adapter does) or keep a native path — and reflect that in the matrix so listed coverage matches what's qualified or dispatched? Happy to help line up the Edge-LLM side.

File: website/docs/getting-started/installation.md [hunk 2]
```diff
@@ -164,3 +164,5 @@ cmake --build build -j
 
 CI and release validation use the wheel path, not the Python-only editable
 install.
+
+{/* Collaborative review anchor. */}
```
Comment (nvluxiaoz) [inline website/docs/getting-started/installation.md:168]: **[Claude Agent Review]** The docs mention Jetson Thor qualification ("hardware-qualified Jetson Thor Wan2.2," "performance qualification on Jetson AGX Thor") and GB300. I wasn't able to find explicit build instructions for Thor — there are `docker_build_gb300.sh`/`docker_run_gb300.sh`, but no Thor/Jetson build or cross-compile script or documented steps, and the `qualified` profiles in the tree are all x86_64/A100/sm80. Could we add explicit, reproducible build/cross-compile instructions (source → bundle → device) for each platform the project claims — Thor especially? That would let readers substantiate the "hardware-qualified Jetson Thor" wording.

File: website/docs/features/quantization.md [hunk 3]
```diff
@@ -215,3 +215,5 @@ Useful task-specific checks include logits and generated text for decoders,
 image/video health and reference comparisons for diffusion, waveform or ASR
 checks for audio, and numerical error metrics for time-series models. A generic
 unit test of the shared format core is not model qualification.
+
+{/* Collaborative review anchor. */}
```
Comment (nvluxiaoz) [inline website/docs/features/quantization.md:231]: **[Claude Agent Review]** For quantization, would it be worth starting from NVIDIA ModelOpt as the foundation rather than the hand-built Q/DQ graph stack? You already use `mtq.quantize` for calibration, and ModelOpt covers the same formats (plus KV-cache/visual quant) on the supported NVIDIA path, which would reduce divergence from the rest of the TensorRT ecosystem. Could you share the rationale for the custom graph construction — I'd like to understand the tradeoff before release.

File: README.md [hunk 4]
```diff
@@ -269,3 +269,5 @@ general correctness or compatibility guarantee.
 | `xlm-roberta-base` | 🟢 Green |
 | `xlnet-base` | 🟢 Green |
 | `z-image-turbo` | 🔴 Red |
+
+<!-- Collaborative review anchor. -->
```
Comment (nvluxiaoz) [inline README.md:321]: **[Claude Agent Review]** Small wording suggestion: the "production"/"release" framing ("native production application," "GB300/aarch64 release") may read as more mature than the current state — the qualified surface is 3 x86/A100 profiles, with the runner unpublished, no reproducible perf, and public CI not running tests. Could we soften this to reflect the actual status (e.g., developer preview / declared-not-qualified) so expectations line up?

File: website/docs/getting-started/model-support.md (hunk 1 above)
Comment (nvluxiaoz) [inline website/docs/getting-started/model-support.md]: **[Claude Agent Review]** Could we also make the support-matrix / model-support lists a bit clearer to read? As a reviewer it isn't always obvious per entry what's actually supported vs. declared — e.g., how the 78 family plugins, the 205 E2E manifests, and the small set of qualified profiles relate, and which models a user can actually build and run from this source today vs. discovery/roadmap. A per-row status legend (qualified / candidate / declared-only) plus a short "what you can run from source today" summary would make the matrix much easier to trust. A couple of the rows weren't clear to me on a first read. Thanks!

**Comment by nvluxiaoz** [comment]
**[Claude Agent Review]** Thanks for putting this up for review. A few release-readiness questions, mostly about lining up the docs with what's qualified/dispatched:

1. Could we add explicit per-platform build instructions for every platform claimed (Thor especially — I couldn't find a Thor build path)?
2. Several families (InternVL, Qwen3.5, Qwen-VL, Phi-4-Multimodal, Nemotron-Omni, Qwen3-Omni) overlap TensorRT Edge-LLM's qualified implementations — could we clarify per family whether they dispatch to Edge-LLM or keep a native path, and reflect that in the matrix?
3. Would starting quantization from NVIDIA ModelOpt make sense (rather than a parallel Q/DQ stack)?
4. Could we reconcile the "hardware-qualified Thor / GB300 / production" wording with the current qualified surface (3 x86/A100 profiles)?

I've also left a clarity request on the support-matrix itself (inline on model-support). Happy to discuss any of these — thanks!

**Comment by yizhuoz004** [comment]
I found the "Getting Started" section still a bit too wordy, as a user I was expecting a very concise page that lists commands from installation, build and execute in a single page. and then other pages may explain more details of every step. Now there are ~10 steps in 3 pages before user actually sees an example.

File: website/docs/getting-started/model-support.md (hunk 1 above)
Comment (yifeif-nv) [reply website/docs/getting-started/model-support.md]: Intended strategy is for these platform that edge-llm already have support we'll do the dispatch. Although we are not expecting all the model dispatch to be finished by the OSS release. I'll clearly document this and mention that the full dispatch for all model will be roll out in batches in the first few nightlys

File: website/docs/getting-started/installation.md (hunk 2 above)
Comment (yifeif-nv) [reply website/docs/getting-started/installation.md:168]: Good catch! I will add this and add a THOR Docker.

File: README.md (hunk 4 above)
Comment (yifeif-nv) [reply README.md:321]: Yeah, makes sense. I will change the wording to gear it more toward a set of reference implementations.

File: website/docs/getting-started/installation.md (hunk 2 above)
Comment (yifeif-nv) [reply website/docs/getting-started/installation.md:168]: Addressed in the latest commit

File: website/docs/getting-started/model-support.md (hunk 1 above)
Comment (yifeif-nv) [reply website/docs/getting-started/model-support.md]: Addressed in the latest commit

File: README.md (hunk 4 above)
Comment (yifeif-nv) [reply README.md:321]: Addressed in the latest commit

File: website/docs/features/quantization.md (hunk 3 above)
Comment (yifeif-nv) [reply website/docs/features/quantization.md:231]: The quantization strategy in TensorRT-Model-Connect is the following:

1. First, we will run through the Modelopt recipe (or any other available quantization recipe).
2. Based on the recipe, we will ask the agent to build a QDQ graph so that we can avoid all the exporting and importing pain. And this QDQ graph is being treated as a specialization.
3. Then, we will be caching the quantization scales in the repo so the user does not need to run calibration again on their end.

We do offer an experimental Modelopt integration directly, but that's not strictly required. 

Please let me know if this addresses your concern.

File: website/docs/getting-started/model-support.md (hunk 1 above)
Comment (yifeif-nv) [reply website/docs/getting-started/model-support.md]: Address in the latest comment. So everything in the big table at the bottom is what we are testing in the CI. We've also added a section explaining that anything resolving to the same model ID on Hugging Face should be able to run, even though we are not testing every single fine-tuned model.

**Comment by yifeif-nv** [comment]
> I found the "Getting Started" section still a bit too wordy, as a user I was expecting a very concise page that lists commands from installation, build and execute in a single page. and then other pages may explain more details of every step. Now there are ~10 steps in 3 pages before user actually sees an example.



> **[Claude Agent Review]** Thanks for putting this up for review. A few release-readiness questions, mostly about lining up the docs with what's qualified/dispatched:
> 
> 1. Could we add explicit per-platform build instructions for every platform claimed (Thor especially — I couldn't find a Thor build path)?
> 2. Several families (InternVL, Qwen3.5, Qwen-VL, Phi-4-Multimodal, Nemotron-Omni, Qwen3-Omni) overlap TensorRT Edge-LLM's qualified implementations — could we clarify per family whether they dispatch to Edge-LLM or keep a native path, and reflect that in the matrix?
> 3. Would starting quantization from NVIDIA ModelOpt make sense (rather than a parallel Q/DQ stack)?
> 4. Could we reconcile the "hardware-qualified Thor / GB300 / production" wording with the current qualified surface (3 x86/A100 profiles)?
> 
> I've also left a clarity request on the support-matrix itself (inline on model-support). Happy to discuss any of these — thanks!

I think all the concerns have been addressed in the latest commit. Please re-review the README doc and see if that helps with the understanding. For the quantization, what ModelOpt provides is a set of scalesthat can be used in the QDQ node. So we are actually not using a separate QDQ stack, but we are getting all those scales from ModelOpt directly. We are just avoiding the ONNX exporting and importing cost.

**Comment by yifeif-nv** [comment]
> I found the "Getting Started" section still a bit too wordy, as a user I was expecting a very concise page that lists commands from installation, build and execute in a single page. and then other pages may explain more details of every step. Now there are ~10 steps in 3 pages before user actually sees an example.

In the latest revision, I've reorganized the quickstart part a little bit. My hope is the majority of users will just go with the agentic quickstart and won't need to go through the manual build steps. Let me know if the current form factor looks better to you now.

File: README.md [hunk 5]
```diff
@@ -96,49 +125,230 @@ encoder/embedding/reranking models, translation, vision-language and OCR,
...
+the declared Python execution profiles. Start with the repository already
+cloned and a terminal open at the checkout root. The host must have a
+working NVIDIA driver, Docker Engine with NVIDIA Container Toolkit support,
+network access, and enough disk space to build the image and model bundle. No
+other host directory is assumed.
 
-Use [Getting Started](website/docs/getting-started/overview.md) for
-prerequisites and the complete first-inference path. The source-build outline
-is:
+Build the development image for one target SM. `TORCH_CUDA_ARCH_LIST` uses the
```
Comment (xuanzic) [inline README.md:144]: Few suggestions based on recent development on non-gb300/aarch64 instance in priorities order:

in the installation session, we can add one supported x86_64 example, including something like -DCMAKE_CUDA_ARCHITECTURES=89 for other gpu types.
Potential TensorRT version collision: The base container exposed a different system version from the intended latest TRT SDK. Without explicit paths, CMake could skip the backend or link against the wrong cohort. Suggestion could be Document explicit TRTMC_TRT_INCLUDE_DIR , TRTMC_TRT_LIBRARY, plus a post-build ABI check.
Document Suggested disk, GPU-memory, workspace and build-time requirements. For example, Qwen-VL bundle was about 15 GB and needed a 12 GiB TensorRT builder workspace. This affects disk capacity, build time, and container mounts.

File: website/docs/getting-started/build-and-run.md
Comment (xuanzic) [inline website/docs/getting-started/build-and-run.md:39]: Explain the flags like max_prefill_length, max_new_tokens, and max_cache_length and how they interact so that user can build the bundle correctly.

Comment (chaofengw-nv) [inline website/docs/getting-started/build-and-run.md]: it seems that all  ".trtfb" have been renamed to ".bundle", ref: https://github.com/NVIDIA/TensorRT-Model-Connect/pull/839， Should we update all related desc?

Comment (yifeif-nv) [reply website/docs/getting-started/build-and-run.md]: Yup just updated everything to .bundle
